shell/synchronize/etc/datf-share.cfg

```diff
@@ -1,6 +1,6 @@
-source_dir="/groups/umcg-solve-rd/scr01/datf/${data_group}/share"
+source_dir="groups/umcg-solve-rd/datf/${data_group}/share"
```

Comment:
Are you sure the / at the front and changing the absolute into a relative path needs to be removed?

Reply:
Never mind, you are now using rsync module syntax, which explains why the leading / had to go.